contents/blog/how-to-pivot.md

```diff
@@ -0,0 +1,214 @@
+---
+date: 2021-01-14
+title: A story about pivots
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+PostHog has pivoted a _lot_.
```

Comment:
nit: I'd emphasize the 'a' as well given that that's an expression used together